yoyodyne/datasets.py

```diff
@@ -59,21 +59,19 @@ def __init__(
             filename (str): input filename.
             tied_vocabulary (bool): whether the source and target should
                 share a vocabulary.
-            source_col (int, optional): 1-indexed column in TSV containing
+            source_col (int): 1-indexed column in TSV containing
                 source strings.
-            target_col (int, optional): 1-indexed column in TSV containing
+            target_col (int): 1-indexed column in TSV containing
                 target strings.
-            source_sep (str, optional): separator character between symbols in
-                source string. "" treats each character in source as a symbol.
-            target_sep (str, optional): separator character between symbols in
-                target string. "" treats each character in target as a symbol.
+            source_sep (str): separator character between symbols in source
+                string. "" treats each character in source as a symbol.
+            target_sep (str): separator character between symbols in target
+                string. "" treats each character in target as a symbol.
             **kwargs: ignored.
         """
         if source_col < 1:
             raise Error(f"Invalid source column: {source_col}")
         self.source_col = source_col
-        if target_col == 0:
```

Comment:
I think this logging should still be present (unless we move it to `train`)

Reply:
It's already logged in trainer, which is why I got rid of it.